docs/index.md

```diff
@@ -19,8 +19,16 @@ You can also preview the capabilities of the toolkit by downloading the [Windows
 
 Feel free to browse the documentation using the table of contents on the left side of this page.
 
-## Open Source
+# Open Source

```

Comment:
Documents should only have a single H1

Reply:
Using a single `#` indicates that it is a H1. This change contradicts the comment.

Reply:
Sorry, I'm saying this change from the current H2, to an H1 is incorrect

Reply:
Sorry, I might be being confused by the UI. It looked like you'd proposed this as a change, not that you were commenting on the change that was made.

Reply:
Just to make it more confusing, I just added a suggestion to change it back now 😄